subcommand/lifecycle-sidecar/command.go

```diff
@@ -142,10 +143,6 @@ func (c *Command) validateFlags() error {
 	if err != nil {
 		return fmt.Errorf("-consul-binary %q not found: %s", c.flagConsulBinary, err)
 	}
-	logLevel := hclog.LevelFromString(c.flagLogLevel)
```

Comment:
No need to validate this here since it's done now via `common.Logger()`

Reply:
We left some of these tests the other commands, e.g. `create-federation-secret`. I think it's useful to keep these tests because they provide additional validation that commands actually check and return the error in the UI.

Reply:
This isn't a test though, this was called in `validateFlags()`:
```go
	err := c.validateFlags()
	if err != nil {
		c.UI.Error("Error: " + err.Error())
		return 1
	}

	logger, err := common.Logger(c.flagLogLevel)
	if err != nil {
		c.UI.Error(err.Error())
		return 1
	}

```

But now that `common.Logger` validates `c.flagLogLevel` it doesn't need to also be validate here.

Reply:
Oh sorry! please ignore my comment then 😄 🤦‍♀️